lib/export_tables_to_big_query.rb

```diff
@@ -76,7 +76,7 @@ def run!
     rescue StandardError => e
       # If any table causes an uncaught error, no data from any later table is sent.
       # Catch errors and skip the failing tables
-      # TODO: alert the team (devs + PA) to any failing tables.
+      Rollbar.error("The table '#{table}' failed during the BigQuery export job.", details: e)
```

Comment:
"The '#{table}' table" would probably be easier to read for my small brain

Reply:
Table table. (Table)